src/main/java/se/sundsvall/casemanagement/integration/rest/lantmateriet/RegisterbeteckningClient.java

```diff
@@ -5,22 +5,21 @@
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestParam;
 
 import se.sundsvall.casemanagement.integration.rest.lantmateriet.configuration.LantmaterietConfiguration;
 import se.sundsvall.casemanagement.integration.rest.lantmateriet.model.Registerbeteckningsreferens;
 
 import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
 
-//@Path("/v4")
+//@Path("/v5")
```

Comment:
Behöver den här raden uppdateras öht om den nu ändå är bortkommenterad?

Reply:
Tänkte att det var rimligt att hålla det up to date. Men det är nog ännu rimligare att ta bort den helt.

Reply:
Känns som ett rimligt resonemang